include/simulation_lifecycle/error.h

```diff
@@ -30,4 +30,11 @@
 /** VISUALIZATION ERRORS 400-499 */
 // TODO
 
+/** CONVERSION ERRORS 500-599 */
+#define INPUT_PATH_EMPTY 500
+#define INPUT_PATH_FOLDER_ERROR 501
+#define FILE_FORMAT_INCORRECT 502
```

Comment:
Can you add the `CONVERT` prefix to all the errors? In this way, we can know that the error has to do with `CONVERT`

Reply:
Thank you @romancardenas for the review! I've made the changes and just pushed a new commit.